packages/backend/src/tenants/settings/model.ts

```diff
@@ -74,3 +75,23 @@ export const formatSettings = (
   }
   return settingsObj
 }
+
+const validateUrlTenantSetting = (url: string): boolean => {
+  try {
+    return !!new URL(url)
+  } catch (err) {
+    return false
+  }
+}
+
+const validateNumberTenantSetting = (numberString: string): boolean => {
+  return !!(Number.isFinite(Number(numberString)) && Number(numberString) > 0)
```

Comment:
Maybe we can have a setting that allows 0? 
For example, `WEBHOOK_MAX_RETRY` seems like it should be able to accept 0